qcodes/instrument_drivers/american_magnetics/AMI430.py

```diff
@@ -238,6 +238,10 @@ def __init__(
                                'heating switch': 9,
                                'cooling switch': 10,
                            })
+        self.add_parameter('ramping_state_check_interval',
+                           initial_value=0.05,
+                           unit="s",
+                           vals=Numbers(0, 10))
```

Comment:
```suggestion
                           vals=Numbers(0, 10),
                           set_cmd=None)
```
This needs a set_cmd to allow it to be settable